server/routes/index.ts

```diff
@@ -30,6 +31,11 @@ export default function routes(service: Services): Router {
       // sort: ['eventDateAndTime,ASC'],
     })
 
+    // No authorisation at this point, no data shown in production
+    if (config.environment === 'prod') {
```

Comment:
FYI: @jonesj2193 / @ushkarev I've added the simple logic we discussed about to _**not**_ show data in production. We'll do _something_ similar for the other pages.

Reply:
cool. could throw a 404. doesn't really matter!